test/langtools/jdk/javadoc/doclet/testSnippetTag/TestSnippetMarkup.java

```diff
@@ -370,6 +370,9 @@ public void testStart(Path base) throws Exception {
         testPositive(base, testCases);
     }
 
+    // @replace on a blank line will not do anything bad!
```

Comment:
What is the purpose of this comment? Is this used as snippet source somewhere?

Reply:
This is an artifact of me thinking of three interdependent PRs simultaneously. A marginal note, if you will. This line should be translated into a new test in the upcoming PR, #6461. I'll move it there, thanks.